sagemaker-training-compiler/tensorflow/single_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    os.system(f"aws s3 sync {source} {destn}")
</pre><p>The sync command may be the issue with the CI failure. I recommend trying to download the dataset from sagemaker-sample-files, and then upload to the default bucket, rather than trying this cross-account, potentially cross-region sync.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>